cmd/create/cluster/cmd.go

```diff
@@ -3079,34 +3101,39 @@ func run(cmd *cobra.Command, _ []string) {
 
 	if isSTS {
 		if mode != "" {
-			if !output.HasFlag() || r.Reporter.IsTerminal() {
-				r.Reporter.Infof("Preparing to create operator roles.")
-			}
-			err := operatorroles.Cmd.RunE(operatorroles.Cmd, []string{clusterName, mode, permissionsBoundary})
-			if err != nil {
-				r.Reporter.Errorf("There was a problem creating operator roles: %v", err)
-				os.Exit(1)
+			if len(operatorRoles) == 0 {
```

Comment:
I recall that this is handled as part of the call for creating operator roles command :thinking: . Not sure if required